### PR TITLE
Use new array syntax and make a few minor tweaks

### PR DESCRIPTION
**Pull request by bocharsky-bw**

### CODE REVIEW

File: page_creation.rst [hunk 1]
```diff
@@ -133,9 +133,9 @@ variable so we can render that::
         {
             $number = mt_rand(0, 100);
 
-            return $this->render('lucky/number.html.twig', array(
-                'number' => $number
-            ));
+            return $this->render('lucky/number.html.twig', [
+                'number' => $number,
+            ]);
```
Comment (xabbuh) [inline page_creation.rst]: I am against making this change. Using the short array syntax in newer branches makes merging branches up a nightmare.

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: page_creation.rst (hunk 1 above)
Comment (bocharsky-bw) [inline page_creation.rst]: @xabbuh Ah, it makes sense. So making changes to the new array syntax is pointless in docs? I'm just wondering for myself for the future.

File: page_creation.rst [hunk 2]
```diff
@@ -209,11 +209,11 @@ Bundles are registered in your ``app/AppKernel.php`` file (a rare PHP file in th
     {
         public function registerBundles()
         {
-            $bundles = array(
+            $bundles = [
```
Comment (bocharsky-bw) [inline page_creation.rst]: @xabbuh Could we keep the new array syntax here? We already used it in [Symfony SE v3](https://github.com/symfony/symfony-standard/blob/3.0/app/AppKernel.php#L10-L19).

File: page_creation.rst (hunk 1 above)
Comment (wouterj) [inline page_creation.rst]: @bocharsky-bw yes. I think we should wait till PHP <5.4 is completely dropped in Symfony (that'll be when 2.8 reaches end of maintainance). We can then safely do this change for all code examples in all branches at one go.

File: page_creation.rst (hunk 2 above)
Comment (wouterj) [inline page_creation.rst]: I would say no. If we change the line below this or move the example in some way, we'll always get merge conflicts.

**Comment by wouterj** [comment]
I agree with @xabbuh on the short array syntax, but I like the other changes. 👍 

If you can update this PR to remove the short array syntax, please do so. Otherwise, we'll do while merging.

Status: reviewed

File: page_creation.rst (hunk 1 above)
Comment (bocharsky-bw) [inline page_creation.rst]: Fair enough 👍

File: page_creation.rst (hunk 2 above)
Comment (bocharsky-bw) [inline page_creation.rst]: Thanks for the clarification!

**Comment by bocharsky-bw** [comment]
Short array syntax reverted.

**Review by wouterj — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @bocharsky-bw.

**Review by xabbuh — APPROVED** [review]

**Pull request closed without merging**